netlify.toml

```diff
@@ -0,0 +1,29 @@
+[Settings]
+  ID = "warpr"
+
+# Settings in the [build] context are global and are applied to all contexts unless otherwise overridden by more specific contexts.  
+
+[build]
+# This is the directory to change to before starting a build. 
+  base    = "."
+  # NOTE: This is where we will look for package.json/.nvmrc/etc, not root.
```

Comment:
Nitpick about indentation.

Reply:
@hectcastro This is how it is in their example docs, so I figured it was best practice in TOML. Looking at the official repo, this may not be the case. https://github.com/toml-lang/toml/blob/master/examples/example-v0.4.0.toml.